src/lib/Utils.js

```diff
@@ -20,3 +20,48 @@ export const yearsAgo = (years, date, format = "MMMM YYYY") => {
   let dt = moment(date)
   return date && dt.subtract(years, "years").format(format)
 }
+
+export const displayDateValue = (value) => {
+  return moment(value).format("DD MMMM YYYY")
+}
+
+export const displayEnumValue = (value, items) => {
+  const result = items.find((item) => item.value === value)
+  return result && result.label
+}
+
+export const displayYesNoValue = (value) => {
+  return value ? "Oui" : "Non"
+}
+
+export const displayCurrencyValue = (value) => {
+  return `${value}\xa0€`
+}
+
+export const displayDepcomValue = (codePostal, nom) => {
+  return `${codePostal} (${nom})`
+}
+
+export const displayValue = (value, question, component) => {
+  if (value === undefined) return value
+  let result
+
+  switch (question.questionType) {
+    case "date":
+      result = displayDateValue(value)
+      break
+    case "enum":
+      result = displayEnumValue(
+        value,
+        executeFunctionOrReturnValue(question, "items", component)
+      )
+      break
+    case "number":
+      result = `${value} ${question.unit}`
+      break
+    case undefined:
```

Comment:
weird to have undefined defaulting as a yes no
should we manage a boolean type ? and throw an error if the questionType is unmanaged ?

Reply:
C'est un choix qui a été prit lors de la mutualisation des questions simples. Comme tu peux le voir dans le fichier `IndividuProperties.js` et `MutualizedStep.vue`, une question sans `questionType` (= undefined) est une question yes/no.

Reply:
Je partais également sur l'idée du type "yesno"
Une erreur n'est pas sensé arriver dans ce cas de figure, car la fonction n'est appelé que si il est référencé comme question mutualisée